test/spec/common/client/interface.js

```diff
@@ -20,25 +20,25 @@ test('AlgoliaSearch client API spec', function(t) {
 
   var expectedProperties = [
     'addAlgoliaAgent',
-    'addQueryInBatch',
     'addApiKey',
+    'addQueryInBatch',
     'addUserKey',
     'addUserKeyWithValidity',
     'batch',
-    'updateApiKey',
```

Comment:
I sorted them alphabetically, Ince that seemed to be what they used to be sorted in once